### DISCUSSION TITLE
Generated javascript client with connect-es - How to know if a property is output only?

### DISCUSSION THREAD

**Discussion by danvuong** [connectrpc/connect-es#1020] [Q&A] [answered]
Hello everyone,

I have a JavaScript client generated with connect-es. 
In the proto files I can find if a property is OUTPUT_ONLY or not.

`string title = 2 [(google.api.field_behavior) = OUTPUT_ONLY];`

 But in the generated js files I can't find this information. 

`{ no: 2, name: "title", kind: "scalar", T: 9 /* ScalarType.STRING */ },`

Do you know if there is a way to find information about OUTPUT_ONLY in the generated js files?

Thank you

**Comment by srikrsna-buf** [comment, marked as answer]
Hey! This is related to the runtime [@bufbuild/protobuf](github.com/bufbuild/protobuf-es). It doesn't yet support this but this is actively being worked on: https://github.com/bufbuild/protobuf-es/issues/397

**Reply by danvuong** [reply]
Thank you I will check it!

**Reply by nicksnyder** [reply]
This is now supported in protobuf-es v2, which is used by connect-es v2
